content/en/docs/Rleases/_index.md

```diff
@@ -1,3 +1,10 @@
+---
+title: Releases 
+linkTitle: "Releases"
+weight: 45
+description: Kyverno Release Notes.
```

Comment:
The description does not need a period.

Reply:
I don't understand. Can you please elaborate?

Reply:
The directory where this file is stored is `content/en/docs/Rleases/` where you've misspelled "Releases" (missing an "e").

Reply:
sure, I just corrected that